config/locales/zh-TW.yml

```diff
@@ -55,7 +55,7 @@ zh-TW:
       perform_full_suspension: 進行停權
       profile_url: 個人檔案網址
       public: 公開
-      push_subscription_expires: PuSH 訂閱逾期
+      push_subscription_expires: 推播訂閱過期
```

Comment:
Do you need to translate "PuSH"?

Reply:
It usually gets translated as, simply, "push". I guess it's fine to do so.